### ISSUE TITLE
More mascot appearances

### ISSUE THREAD

**Issue by ineffyble** [mastodon/mastodon#20879]
### Pitch

Make the Mascot show in more places/pages in the Mastodon UI.

### Motivation

The Mastodon mascot is great, and some of the customised ones that other servers have set are also great. But a lot of users and visitors might never see the mascot now that it only appears in the advanced web view (and possibly announcements?).

**Comment by BrushBandit** [comment]
yeah I'd like an option to return the mascot in single column mode and mobile interface as well. hope they bring it back or add more customization options >_<
